test/check-application

```diff
@@ -3086,6 +3087,19 @@ class TestApplication(testlib.MachineCase):
         b.wait_text("button.pf-v5-c-select__menu-item.pf-m-disabled", "No images found")
         b.wait_in_text(".pf-v5-c-alert.pf-m-danger", "couldn't search registry")
 
+        # Searching for full url with tag finds the image
+        b.set_input_text("#run-image-dialog-name", "tagged-busybox")
+        b.set_input_text("#create-image-image-select-typeahead", "localhost:80/my-busybox:my-tag")
+        b.click('button.pf-v5-c-toggle-group__button:contains("All")')
+        b.wait_text("button.pf-v5-c-select__menu-item:not(.pf-m-disabled)", container_name + ":my-tag")
+        b.click(f"button.pf-v5-c-select__menu-item:contains({container_name + ":my-tag"})")
+        b.click("#create-image-create-btn")
+
```

Comment:
Do we already have a test somewhere which checks the behaviour on a registry *with* search? I.e. the localhost:3000 one (or whatever the port was)? There is a lot of if/else for this, and the missing coverage also hints that there isn't -- can you please add one?

Reply:
The missing coverage is for a fallback when there are no configured search registries. And since we always set these it doesn't go to the `else` branch. However I added a check that the `manifest` query works on searchable registry.